public/_data/repositories/delivery.yml

```diff
@@ -0,0 +1,32 @@
+section:
+  description: Tools to ensure your applications have access to the privileged systems they need.
+  categories:
+  - name: Secretless Broker
+    description: Secure your apps by making them Secretless.
+    repos:
+      - name: cyberark/secretless-broker
+        url: https://github.com/cyberark/secretless-broker
+        description: Secretless Broker
+        #thumb: CyberArk_Conjur_logo.svg
+        #image-alt:
+  - name: Summon
+    description: Get your secrets into source code.
```

Comment:
I feel like we should use a different tag line... but I don't have any great suggestions.